content/nginxaas-azure/changelog-archive/changelog-2023.md

```diff
@@ -70,7 +70,7 @@ To see a list of currently active issues, visit the [Known issues]({{< ref "/ngi
    To get started using HTTP/3 and NGINXaaS:
 
    - Update the [network security group](https://docs.microsoft.com/en-us/azure/virtual-network/tutorial-filter-network-traffic#create-security-rules) associated with the NGINXaaS deployment’s subnet to allow inbound traffic for HTTP/3 UDP ports in the NGINX configuration.
-   See our [FAQ]({{< ref "/nginxaas-azure/faq" >}}), for limits on how many unique ports may be specified in a configuration and a list of restricted ports.
+   See the [NGINXaaS architecture]({{< ref "/nginxaas-azure/overview/overview.md#nginxaas-architecture" >}}), for limits on how many unique ports may be specified in a configuration and a list of restricted ports.
```

Comment:
Is this the right section? This one seems to include some limits:

```suggestion
   See the [NGINXaaS limitations]({{< ref "/nginxaas-azure/overview/overview.md#limitations" >}}), for limits on how many unique ports may be specified in a configuration and a list of restricted ports.
```

Reply:
good catch, I moved the limits after I updated this doc